### PR TITLE
Allows subjects of the dominion to speak to eachother

### PR DESCRIPTION
**Pull request by Pontenerd**
## About The Pull Request
Makes etherials able to speak and understand moffic, and moths can understand voltaic

## Why It's Good For The Game

Both races are part of the dominion of light, they should be able to understand one another

## Changelog
:cl:
tweak: changes the languages moths and etherials can understand

/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
No.

**Comment by Pontenerd** [comment]
> No.

Can you elaborate on this further? I've already gotten approval from the lore lead about this

**Comment by BetaCentari** [comment]
> No.

This was made on my request, beginning to implement features based on the written lore that's sat for too long without any attention or code implementations.

**Comment by Karmic-Skink** [comment]
> > No.
> 
> This was made on my request, beginning to implement features based on the written lore that's sat for too long without any attention or code implementations.

Non-canon "lore" will not be integrated into code.
Use common.

**Comment by BetaCentari** [comment]
> > > No.
> > 
> > 
> > This was made on my request, beginning to implement features based on the written lore that's sat for too long without any attention or code implementations.
> 
> Non-canon "lore" will not be integrated into code. Use common.

I am the lore coordinator now and I have decided that this along with much of the written lore on the Dominion of Light is in fact canon. If you disagree then we can discuss this elsewhere.

**Comment by Karmic-Skink** [comment]
> 

The concepts of "Dominion of Light", "Draconic Empire" and "Federation" were rejected shortly after they were proposed due to misaligning with the design goals and intentions of NSV13.

This discussion was had over a year ago. If that hasn't been properly communicated, I suggest you take it up with the former lore coordinator who appears to have failed to communicate that fact.

**Review by Karmic-Skink — DISMISSED** [review]
These require //NSV13 tags as this is a base file

**Comment by Karmic-Skink** [comment]
Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species.
Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.

**Comment by BetaCentari** [comment]
> Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species. Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.

I've only just begun my work as loor coordinator and I'm actively mending those plot holes. I understand the prospects of scrutiny and secrecy though I have been asked to nerf IPC language abilities, a PR will be through shortly. 😄

**Comment by covertcorvid** [comment]
Per our discussions the moth-ethereal connection is supported by lore which has been made canon, so this is fine.
We still won't be making the extra factions into influential entities in gameplay but flavor things like this are alright, and it's only one species pair.

**Comment by Karmic-Skink** [comment]
> > Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species. Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.
> 
> I've only just begun my work as loor coordinator and I'm actively mending those plot holes. I understand the prospects of scrutiny and secrecy though I have been asked to nerf IPC language abilities, a PR will be through shortly. 😄

Personally, I still think an organic being able to read electricity without physical enhancement is dumb.

But yeah, your predecessor basically ignored 99% of non-lore team feedback.

**Comment by Karmic-Skink** [comment]
> > Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species. Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.
> 
> I've only just begun my work as loor coordinator and I'm actively mending those plot holes. I understand the prospects of scrutiny and secrecy though I have been asked to nerf IPC language abilities, a PR will be through shortly. 😄

I've sent Bokkie a series of DMs about how we can potentially implement this (and other stuff) better.
Standby for jankcode.

**Comment by BetaCentari** [comment]
> > > Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species. Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.
> > 
> > 
> > I've only just begun my work as loor coordinator and I'm actively mending those plot holes. I understand the prospects of scrutiny and secrecy though I have been asked to nerf IPC language abilities, a PR will be through shortly. 😄
> 
> I've sent Bokkie a series of DMs about how we can potentially implement this (and other stuff) better. Standby for jankcode.

Originally I wanted to create a separate language though this was recommended instead. Do you think that adding another language for moths and ethereals to have would be a good way to implement this feature? (Along with taking away IPC language rights)

**Comment by BetaCentari** [comment]
> > > > Significant lore plot holes aside, this is effectively both a nerf secrecy and a buff scrutiny for the respective species. Messing with languages, particularly as we already have IPCs with AI level language abilities, is ill advised.
> > > 
> > > 
> > > I've only just begun my work as loor coordinator and I'm actively mending those plot holes. I understand the prospects of scrutiny and secrecy though I have been asked to nerf IPC language abilities, a PR will be through shortly. 😄
> > 
> > 
> > I've sent Bokkie a series of DMs about how we can potentially implement this (and other stuff) better. Standby for jankcode.
> 
> Originally I wanted to create a separate language though this was recommended instead. Do you think that adding another language for moths and ethereals to have would be a good way to implement this feature? (Along with taking away IPC language rights)

DISREGARD, I've been informed of your idea, Skink and I'm so on board for it! Great idea.

**Comment by Karmic-Skink** [comment]
> DISREGARD, I've been informed of your idea, Skink and I'm so on board for it! Great idea.

Yeah, I hope it works. Should be a good way of inserting that sort of stuff in optionally to give people a bit more of a background choice.

File: code/modules/language/language_holder.dm [hunk 1]
```diff
@@ -350,9 +351,17 @@ Key procs
...
 	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
-							/datum/language/voltaic = list(LANGUAGE_ATOM))
+							/datum/language/voltaic = list(LANGUAGE_ATOM),
+							/datum/language/moffic = list(LANGUAGE_ATOM))
+/datum/language_holder/ipc //nsv13
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/machine = list(LANGUAGE_ATOM))
+
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/machine = list(LANGUAGE_ATOM))
```
Comment (Bokkiewokkie) [inline code/modules/language/language_holder.dm:364]: Can you put this in a seperate NSV file and remove the changes to ethereals and moths? This is the only real reason it's testmerged right now and it's going to take a hot minute to PR the changes to the other things properly so I'd rather just merge your PR with the IPC changes.

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Review by covertcorvid — APPROVED** [review]

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**